src/sql/postgres.zig

```diff
@@ -620,6 +620,38 @@ pub const PostgresSQLQuery = struct {
         pending_value.push(globalThis, value);
     }
 
+    pub fn getStatement(this: *PostgresSQLQuery, thisValue: JSC.JSValue, globalObject: *JSC.JSGlobalObject) JSValue {
+        if (this.statement) |statement| {
+            if (PostgresSQLQuery.statementGetCached(thisValue)) |value| {
+                return value;
+            }
+
+            const obj = JSValue.createEmptyObject(globalObject, 2);
+
+            obj.put(globalObject, JSC.ZigString.static("string"), bun.String.init(this.query).toJS(globalObject));
```

Comment:
```suggestion
            obj.put(globalObject, JSC.ZigString.static("string"), bun.String.createUTF8ForJS(globalObject, this.query));
```

Reply:
Perfect, was not aware of this one!

Maybe we should add to the `String.init`'s documentation: "If you are creating a string for JS, you probably want to use createUTF8ForJS"?

Reply:
Actually in this case, I misread and `this.query` is already a `bun.String` so I assume `this.query.toJS(globalObject)` would be more optimal.